test/event_handler_test.py

```diff
@@ -3,24 +3,57 @@
 import io
 import json
 import uuid
+import psycopg2
 from moto import mock_sqs
 from unittest import TestCase
 from unittest.mock import patch
+from datetime import datetime
 
 from src import event_handler
+from src.db_helper import RunInTransaction
 
+EVENT_TYPE = 'session_event'
+TIMESTAMP = '2018-02-10 12:07:32'
+SESSION_EVENT_TYPE = 'success'
 
 @mock_sqs
 class EventHandlerTest(TestCase):
+    __sqs_client = None
+    __queue_url = None
+    db_connection = None
+    db_connection_string = "host='localhost' dbname='postgres' user='postgres' password='secretPassword'"
+
+    @classmethod
+    def setUpClass(cls):
+        cls.db_connection = psycopg2.connect(cls.db_connection_string)
+        with RunInTransaction(cls.db_connection) as cursor:
+            cursor.execute("""
+                CREATE TABLE events (
+                    event_id VARCHAR(40) NOT NULL PRIMARY KEY,
+                    event_type VARCHAR(20) NOT NULL,
+                    timestamp TIMESTAMP NOT NULL,
```

Comment:
Do we need index(es) for the events table?

Reply:
This is only a short lived test DB (it gets killed after the test) so I don't think we need to worry about indexes.

If we want it to match production, then we can add indexes later once we decide what they should be.